lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+
+	attr_reader :menu
+
+	def initialize
+		@menu  = {'Burger'=> 10, 'Pint'=> 4.50,'Steak'=>18,'Wine'=> 22, 'Chips'=> 3.25}
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)